ecmascript/transforms/module/src/common_js.rs

```diff
@@ -155,6 +155,19 @@ where
                             ref specifiers,
                             ..
                         })) => {
+                            // handle: export {sym as alias1, alias2, ...}
+                            for ExportNamedSpecifier { orig, exported, .. } in
+                                specifiers.into_iter().map(|e| match e {
```

Comment:
I think `filter_map` with `None` for other cases is better.
(Although I don't have concrete input which breaks this)

Reply:
I've pushed a change to try to address this.  I hope it's in line with what you were thinking - I had previously used a pattern that I saw was elsewhere in the code.